test/test-genesis.ts

```diff
@@ -1,28 +1,50 @@
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
-import { expect } from "chai";
-import { Contract, utils } from "ethers";
-import { ethers } from "hardhat";
-import { Deployment } from "hardhat-deploy/dist/types";
+/* eslint-disable no-unused-expressions */
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { expect } from 'chai';
+import { Contract, utils } from 'ethers';
+import { ethers } from 'hardhat';
+import { Deployment } from 'hardhat-deploy/dist/types';
 import {
   addLinkFundIfNeeded,
   addressZero,
   deployTestContract,
   mint,
-} from "./test-utils";
+} from './test-utils';
 
-describe("Genesis Contract", () => {
+describe('Genesis Contract', () => {
   let contract: Contract;
   let VRFCoordinatorMock: Deployment;
   let owner: SignerWithAddress;
-  let address1: SignerWithAddress;
   let oracle: SignerWithAddress;
+  let whitelisted: SignerWithAddress;
+  let notWhitelisted: SignerWithAddress;
   const randomNumber = 777;
+  const merkleTreeRoot =
+    '0xe32689cba5d4fa1698e45bcc090b47a13a1ea9c41671c5205bebeecd5b9ebb7f';
```

Comment:
How do you generate these values?

Reply:
check out https://github.com/pxMythics/utils